ledger/ledger-api-client/src/main/scala/com/digitalasset/ledger/client/services/commands/CommandTrackerFlow.scala

```diff
@@ -74,9 +74,9 @@ object CommandTrackerFlow {
                 1,
                 { case e =>
                   logger.warn(
-                    s"Completion Stream failed with an error. Trying to recover in ${backOffDuration} ..",
-                    e,
+                    s"Completion Stream failed with an error. Trying to recover in ${backOffDuration} .."
                   )
+                  logger.debug(e)
```

Comment:
is it worth adding some text to make it clear that this belongs to the log statement above or is the logger + potential trace contexts enough?

Reply:
I'd be enclined to say that logger context + stack trace + the fact that this will appear right after the WARN + the message in the exception itself should be enough, but I'm not going to be the one reading these logs, so I'm open to suggestions. A simple solution could be to just repeat the WARN message in the DEBUG logs:

```suggestion
                  logger.debug(
                    s"Completion Stream failed with an error. Trying to recover in ${backOffDuration} ..",
                    e,
                  )
```

Reply:
your choice, “right after” can be a bit of a tricky concept once you add concurrency but probably clear enough.